.bashrc~

```diff
@@ -0,0 +1 @@
+export CALENDAR_API_KEY="AIzaSyAQNtP4Lcht6mJPGPK4h2Up51v6tGhq3_A"
```

Comment:
this file also has an API key....can you remove it?

Reply:
The API key was replaced as soon as it was exposed (so this one is invalid) but I will make sure to remove it regardless